resources/less/hakija.less

```diff
@@ -134,7 +134,7 @@
   }
 }
 .application__wrapper-element--border {
-  border-top: solid #D7D7D7 1px;
+  border-top: solid 1px #e6e6e6;
```

Comment:
Does this work? Should it be `1px solid #e6e6e6`? Also would prefer to use `@review-separator-color` here.